### DISCUSSION TITLE
Syntax error in --bootset description in Command Details?

### DISCUSSION THREAD

**Discussion by dshaw619** [gitbls/sdm#58] [Q&A] [answered]
The description of the --bootset switch on the Command Details Wiki page (https://github.com/gitbls/sdm/wiki/Command-Details) says "--bootset key:value,key:value,... " but isn't the correct syntax "--bootset key=value,key=value,..."?

**Comment by tiefenauer** [comment]
No, I thought the same because it would then match the syntax in 1pibiot.conf. Also, it's counter-intuitive to me that `1` seems to mean `disabled` and `0` means enabled...

**Comment by gitbls** [comment, marked as answer]
sdm allows both _key:value_ and _key=value_ for `--bootset`.  Command Detail wiki page updated to reflect this.
```
 IFS=":="
```
re @tiefenauer: If you're referring to the values for bootset, such as `powerled`, those values are specified by `raspi-config`.

**Reply by dshaw619** [reply]
Thanks.  Was confusing because the Command Details page and the Bootset and 1piboot page showed different syntax.
